plugins/secretstores/docker/docker_test.go

```diff
@@ -0,0 +1,178 @@
+package docker
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSampleConfig(t *testing.T) {
+	plugin := &Docker{}
+	require.NotEmpty(t, plugin.SampleConfig())
+}
+
+func TestInitFail(t *testing.T) {
+	tests := []struct {
+		name     string
+		plugin   *Docker
+		expected string
+	}{
+		{
+			name:     "invalid id",
+			plugin:   &Docker{},
+			expected: "id missing",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			err := tt.plugin.Init()
+			require.ErrorContains(t, err, tt.expected)
+		})
+	}
+}
+
+func TestPathNonExistant(t *testing.T) {
+	plugin := &Docker{
+		ID:   "default_test",
+		Path: "non/existent/path",
+	}
+	err := plugin.Init()
+	require.Error(t, err)
```

Comment:
Can you please check either for the error type (`require.ErrorIs`) or the error message to make sure we fail at the right spot!?